lblod/frontend-reglementaire-bijlage
# mandatee-table plugin: remove insertion constraint on mandatee table insertion

## Overview
This (draft) contains an update to the editor-plugins dev-commit of https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/469

##### connected issues and PRs:
https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/469

### How to test/reproduce
- Start the app
- Open a decision template
- Notice that you can insert a mandatee table anywhere
- You should also be able to insert mandatee tables in snippets

### Checks PR readiness
